src/v3/test/integration/authenticator-apple-sso-extension.test.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import identify from '@okta/mocks/data/idp/idx/identify.json';
+import withCredentialMock from '@okta/mocks/data/idp/idx/identify-with-apple-credential-sso-extension.json';
+import noCredentialMock from '@okta/mocks/data/idp/idx/identify-with-no-sso-extension.json';
+import stepUpError from '@okta/mocks/data/idp/idx/error-401-okta-verify-apple-sso-step-up.json';
+import { setup } from './util';
+
+const verifyUrl = 'http://localhost:3000/idp/idx/authenticators/sso_extension/transactions/123/verify?\
```

Comment:
Redact?

Reply:
The actual challengeRequest value is material to the flow working. (I believe it's an encoded JSON string but doesn't seem to be b64)